Docs/colors.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Colors | LightUI</title>
+
+    <link rel="stylesheet" href="doc.css">
+</head>
+
+<body>
+    <div class="nav flex-space-between padding-xxl shadow-sm">
+        <!-- <ion-icon class="header-icon" name="sunny-outline"></ion-icon> -->
```

Comment:
you can remove the commented code.

Reply:
This icon is related to switching between dark and white mode. As I haven't added the functionality, I just commented it out.